spec/unit/logstash_output_spec.rb

```diff
@@ -88,105 +70,94 @@
       end
     end
 
-    context "self identity" do
```

Comment:
are the SSL-identity and SSL-trust behaviours no longer specified, or are they now _assumed_ to work simply because we include the HTTP Client mixin? I would rather we continue to specify this behaviour in this plugin, as a safeguard.

Reply:
Ah 🤦 http-mixin holds _generic_ SSL configs and the behaviour depends on who (client? server?) is manipulating it.
I had to remove the configs to align on http-mixin but not unit tests which checks the output behaviour. Thanks for catching up!